greeneeds/projects/views.py

```diff
@@ -79,14 +80,33 @@ def get(self, request):
           except Project.DoesNotExist:
                return JsonResponse({'message' : 'PROJECT_DOES_NOT_EXIST'}, status = 400)
 
-class UserView(View):
+class ProjectUploadView(View):
      @login_decorator
-     def get(self, request):
-          user = request.user
+     def post(self, request):
+          try: 
+               file           = request.FILES.get('formData', None)
+               upload_img_url = FileUpload(s3_client).upload(file)
```

Comment:
객체를 생성하지 않고 바로 해당 method까지 바로 진행할 경우 실용성이 많이 떨어집니다.
다시 객체가 필요할 경우 객체를 다시 할당 해야 되고, 다시 할당된 객체는 처음에 사용했던 객체의 정보를 가지고 있지 않습니다.

```suggestion
               s3_client      = FileUpload(s3_client)
               upload_img_url = s3_client.upload(file)
```